app/dashboard/templates/dashboard/sponsors.html

```diff
@@ -172,38 +176,33 @@
                       </div>
                       <div role="tablist">
                         <b-card no-body class="my-3" v-for="(prize, prizeIndex) in prizes" v-if="prizes">
-                          <b-card-header header-tag="header" class="row mx-0 px-4 py-2 bg-white" role="tab">
+                          <b-card-header header-tag="header" class="justify-content-between row mx-0 px-4 py-2 bg-white" role="tab">
 
                             <div class="col-12 col-md-5 col-lg-6 text-center text-md-left pl-md-0">
                               <div class="my-2">
                                 <p class="m-0">
                                   <a target="_blank" :href="prize.url" class="font-body font-weight-semibold d-inline-block text-black">
                                     [[`${prize.title.slice(0, 100)}${prize.title.length > 100 ? '...' : ''}`]]
                                   </a>
+                                  <span v-if="prize.value_eth" class="tag token inner-tooltip">[[prize.value_eth | decimals]] ETH</span>
+                                  <span v-if="prize.values_usdt" class="tag usd inner-tooltip">[[prize.values_usdt]] USD</span>
                                 </p>
-
                               </div>
                             </div>
 
-                            <div class="col-md-2 my-auto">
-                              <p class="mb-3 m-md-0 text-center">
-                                <span v-if="prize.value_eth" class="tag token">[[prize.value_eth | decimals]] ETH</span>
-                                <span v-if="prize.values_usdt" class="tag usd">[[prize.values_usdt]] USD</span>
-                              </p>
-                            </div>
-
                             <div class="col-md-5 col-lg-4 my-auto font-body font-weight-semibold text-center">
-                              <p v-if="!prize.total_projects" class="font-body font-weight-semibold m-0">No Submissions</p>
+                              <p v-if="!prize.total_projects" class="font-body font-weight-semibold m-0 text-left">No Submissions</p>
 
                               <p v-else class="m-0 position-relative">
-                                <span>
+                                <span style="left: 0;" class="position-absolute">
                                   [[ prize.total_submitted ]] / [[prize.total_projects]] Submitted
                                 </span>
                                 <span v-if="prize.paid" class="ml-5 gc-text-green"><span class="fas fa-check mr-2"></span>PAID</span>
-                                <a v-else :href="`/issue/payout?pk=${prize.pk}`" class="ml-2 ml-xl-5 btn btn-outline-primary font-smaller-5">PAY</a>
+                                <a v-else :href="`/issue/payout?pk=${prize.pk}`" class="ml-5 gc-text-blue">UNPAID</a>
```

Comment:
Why was this changed back ? Could we keep it to what it was

Reply:
@PixelantDesign request me change this to the previous implmementation

Reply:
@PixelantDesign waiting on confirmation ^

Reply:
What  was the change y'all are referring to?

Reply:
@zoek1 looks like it was never supposed to be a link in the first place!
If that is the case -> this makes sense but you'll have to remove the <a tag>

I leave it to you and alisa to decide if on clicking this label -> it behaves as an accordion 

![image](https://user-images.githubusercontent.com/5358146/89785751-7ffa8f00-db38-11ea-8be6-3fdac2c39cc3.png)